build_sm120/tvm/3rdparty/tvm-ffi/libbacktrace/install-debuginfo-for-buildid.sh

```
@@ -0,0 +1,65 @@
+
+sed=/usr/bin/sed
+awk=mawk
+grep=/usr/bin/grep
+objcopy=objcopy
+readelf=readelf
+mkdir_p="/usr/bin/mkdir -p"
+
+build_id_dir="$1"
+src="$2"
+
+buildid=$(LANG=C $readelf -n $src \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)